### PR TITLE
mrc-1986 restyle

### PR DESCRIPTION
**Pull request by hillalex**
Restyles the app to be more similar to the prototype, with baseline settings remaining accessible at the top of the page.

### CODE REVIEW

**Comment by hillalex** [comment]
This is ready for review, but I'd prefer https://github.com/mrc-ide/mint/pull/67 to be merged first because there will be conflicts and I think they'll be easier to resolve that way round!

File: src/app/static/src/app/actions.ts [hunk 1]
```diff
@@ -30,6 +31,15 @@ const currentRegionBaseline = (state: RootState) => {
 
 export const actions: ActionTree<RootState, RootState> = {
 
+    async [RootAction.SetCurrentRegionBaselineSettings](context, payload: DynamicFormData) {
+        const {dispatch, commit} = context;
+        commit(RootMutation.SetCurrentRegionBaselineSettings, payload);
```
Comment (hillalex) [inline src/app/static/src/app/actions.ts:36]: we were previously just committing the settings to the store, then fetching data on "next". but now, if the baseline settings are open while you're viewing figures, we need to fetch fresh data in real time as settings are changed, hence this new action

**Review by EmmaLRussell — COMMENTED** [review]
This looks good. The only thing that still seems a bit weird to me is that, once you move onto the next step, and 'Set up baseline' collapses, it kind of looks like Setup baseline is the heading for the whole page. I think this is why I'm a bit more inclined to keep a title for the second step as well. But I think we can try this as is and get opinions other than mine.

**Comment by hillalex** [comment]
> This looks good. The only thing that still seems a bit weird to me is that, once you move onto the next step, and 'Set up baseline' collapses, it kind of looks like Setup baseline is the heading for the whole page. I think this is why I'm a bit more inclined to keep a title for the second step as well. But I think we can try this as is and get opinions other than mine.

Fair enough, whereas I'm reluctant to add another heading and take up more space! So lets put this out and see what feedback we get :)

**Pull request merged**